API.md

```diff
@@ -268,3 +269,27 @@ instead return a [Joi ValidationError object](https://joi.dev/api/?v=17.9.1#vali
 ```
 
 - Note: If the archive doesn't exist, this will return an empty array
+
+### GET `/archive/:archiveId/payer-account-storage
+
+- Headers: Authorization: Bearer \<JWT from FusionAuth>
+
+- Response
+
+```
+{
+  accountSpaceId: string,
+  accountId: string,
+  spaceLeft: string,
+  spaceTotal: string,
+  filesLeft: string,
+  filesTotal: string,
```

Comment:
I can see an argument that these should be numbers instead, as they do represent a real thing that is numeric. My reasoning for leaving them strings is that this endpoint isn't doing any math on them, it's just retrieving them for display. Further, Javascript numbers are floating point by default, so reliably maintaining their precision involves some extra overhead.